src/transformers/models/mamba/modeling_mamba.py

```diff
@@ -647,12 +650,14 @@ def forward(
         """
         return_dict = return_dict if return_dict is not None else self.config.use_return_dict
 
+        use_cache = kwargs.get("use_cache", None)
```

Comment:
```suggestion
use_cache = use_cache if use_cache is not None else config.use_cache
```
since you added use cache it is